### DISCUSSION TITLE
Anyone would like to help to create discussion forms?

### DISCUSSION THREAD

**Discussion by louislam** [louislam/dockge#144] [Feature Request]
Discussion Template Docs:
https://docs.github.com/en/discussions/managing-discussions-for-your-community/syntax-for-discussion-category-forms

Basically I want to make something like:
https://github.com/louislam/uptime-kuma/issues/new?assignees=&labels=help&projects=&template=ask-for-help.yaml
https://github.com/louislam/uptime-kuma/issues/new?assignees=&labels=feature-request&projects=&template=feature_request.yaml

**Comment by Lord0fBytes** [comment]
I would love to help out with this. I am new to this, but I will try to get this started today.

**Reply by louislam** [reply]
Thank you so much!

**Comment by Lord0fBytes** [comment]
I didnt add a label for the ask-for-help. I wasn't sure if you wanted to keep them without labels or create a new label for those. Just let me know.

**Reply by louislam** [reply]
It throws a error if label is empty. I just added back and perfect. Thanks!

https://github.com/louislam/dockge/commit/9e8bccbf2f9955cd49c7b647cad60d7a237eddf1
